### PR TITLE
Add Exceptions for WP Engine's "Autoupdater" Plugin

### PR DESCRIPTION
**Pull request by AlanBreck**
Adds exceptions to accomodate stylistic differences in WP Engine's `autoupdater` plugin.

### CODE REVIEW

**Comment by AlanBreck** [comment]
I'm going to add the most recently suggested exceptions and push to this PR.

**Comment by AlanBreck** [comment]
Additional exceptions [recommended by @fmarier](https://github.com/brave/security/issues/91#issuecomment-597369704) added. PR ready for review.

File: BraveMinimum/ruleset.xml [hunk 1]
```diff
@@ -79,9 +98,14 @@
...
 		<exclude name="Squiz.Strings.DoubleQuoteUsage.NotRequired"/>
 		<exclude name="Squiz.WhiteSpace"/>
+		<exclude name="Squiz.Operators.ValidLogicalOperators.NotAllowed" />
+		<exclude name="Squiz.PHP.EmbeddedPhp.SpacingAfterOpen" />
+		<exclude name="Squiz.PHP.DisallowMultipleAssignments.FoundInControlStructure" />
+		<exclude name="Squiz.PHP.NonExecutableCode.ReturnNotRequired" />
+		<exclude name="Squiz.Classes.SelfMemberReference.NotUsed" />
 		<exclude name="WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound"/>
 		<exclude name="WordPress.Arrays.ArrayDeclarationSpacing.NoSpaceAfterArrayOpener"/>
-                <exclude name="WordPress.Arrays.ArrayDeclarationSpacing.NoSpaceBeforeArrayCloser" />
```
Comment (fmarier) [inline BraveMinimum/ruleset.xml:84]: Looks like an accidental whitespace change.

**Review by fmarier — APPROVED** [review]
Nit: This file might be easier to read/maintain if we keep the lines in alphabetical order.

**Comment by AlanBreck** [comment]
Thank you for the review! I've fixed the whitespace and sorted the lines.

@fmarier unless you'd like to review those modifications, I can go ahead and merge.

**Comment by fmarier** [comment]
Thanks @AlanBreck !

**Pull request merged**